src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_help.py

```diff
@@ -671,4 +666,7 @@
 -   name: Add a firewall filter rule to accept traffic from the IP mask 127.0.0.0/31.
     text: >
         az iot hub update --name MyIotHub --add properties.ipFilterRules filter_name=test-rule action=Accept ip_mask=127.0.0.0/31
+-   name: Update metadata for an IoT hub.
+    text: az iot hub update --set {set} --name MyIotHub
```

Comment:
This is not a useful example. There should not be a placeholder for {set} but actual valid set syntax.